internal/iam/repository/eventsourcing/model/iam_test.go

```diff
@@ -25,31 +25,31 @@ func TestProjectRoleAppendEvent(t *testing.T) {
 		{
 			name: "append set up start event",
 			args: args{
-				event: &es_models.Event{AggregateID: "AggregateID", Sequence: 1, Type: IAMSetupStarted, ResourceOwner: "OrgID"},
+				event: &es_models.Event{AggregateID: "AggregateID", Sequence: 1, Type: IAMSetupStarted, ResourceOwner: "GrantedOrgID"},
```

Comment:
why did this code change? the test doesn't validate resource owner

Reply:
also in the test cases below

Reply:
I think this was a refactoring problem i didnt realize